ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -100,11 +101,12 @@ class LocalAssemblerData
                                  sm.detJ * wp.getWeight();
 
             // Darcy velocity only computed for output.
-            auto const darcy_velocity = -(k * sm.dNdx *
```

Comment:
The problem why the old code was "insane" according to clang is probably that the minus is not inside the parentheses that are being `eval()`'d.